spec/paper_trail_scrapbook/life_history_spec.rb

```diff
@@ -120,6 +120,40 @@ def target.trailed_related_content
           expect(subject).to match(/author: Dr\. Seuss\[\d+\]/)
         end
       end
+
+      context 'recent first' do
+        before do
+          author
+          book
+          target
+
+          config = PaperTrailScrapbook.config
+          config.recent_first = true
```

Comment:
Ok, this is the right line - little concerned that spec passes if this is commented out.

Reply:
the spec is not checking the sequence - let me see if I can encourage that. the data is limited